spec/muffin_man/listings_spec.rb

```diff
@@ -52,4 +52,22 @@
       expect(JSON.parse(response.body)).to eq(put_listing_result)
     end
   end
+
```

Comment:
Can we also add specs for case like - 
1. seller_id, sku combination not found
2. Cover - issue_locale parameter